### DISCUSSION TITLE
Welcome to obsidian-book-search-plugin Discussions!

### DISCUSSION THREAD

**Discussion by anpigon** [anpigon/obsidian-book-search-plugin#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarcoZubieta** [comment]
My name is Marcvit and I am very happy to have found this community! I am sure that this will be a favorite place to share knowledge, books and code for the improvements that always bring quality to the project for the benefit of all. I am currently looking for research in a wide variety of topics, always from an interdisciplinary approach. I am interested in topics of Applied Ecology, Applied Ontology, Digital Fabrication, New Materials, Ecological Materials, Knowledge Management, Data Science, International Affairs and Cultural Studies .
